core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -6,8 +6,7 @@
 
 {% block header_js %}
   {{ super() }}
-  {% if exploration_version %}
-    <script type="text/javascript">
+    <script ng-if="getExplorationVersion()" type="text/javascript">
```

Comment:
Deindent this and the next few lines by 2.

Also, what happens if you remove the ng-if entirely? I'm not actually sure it's needed...